applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1616,6 +1620,7 @@ get_mailbox_profile(Data, Call) ->
                     ,account_db = AccountDb
                     ,media_extension = kzd_voicemail_box:media_extension(MailboxJObj)
                     ,forward_type = ?DEFAULT_FORWARD_TYPE
+                    ,oldest_message_first = kz_json:is_true(<<"oldest_message_first">>, MailboxJObj, 'false')
```

Comment:
Once you migrate the functions over in `kzd_vmboxes` change this to `kzd_vmboxes:oldest_message_first(MailboxJObj, 'false')`

Reply:
@jamesaimonetti this has been updated. Please review :)